packages/core/src/components/Row/index.tsx

```diff
@@ -65,13 +69,18 @@ export class Row extends React.Component<Props & WithStylesProps> {
           topline && styles.row_topline,
         )}
       >
-        {before && (
-          <div className={cx(compact ? styles.before_compact : styles.before)}>{before}</div>
-        )}
-
-        <div className={cx(styles.primary, truncated && styles.primary_truncated)}>{children}</div>
-
-        {after && <div className={cx(compact ? styles.after_compact : styles.after)}>{after}</div>}
+        {before && <div className={cx(styles.before, inline && styles.inline)}>{before}</div>}
```

Comment:
I don't believe we should remove the compact padding in this PR. Also seems weird to only have 8px padding on spacious (28px) rows.

Reply:
the 8px is to get the column spacing to be consistent with other column spacing in Grid and AdaptiveGrid. i can do it in a separate PR or keep it here

Reply:
Should go through design approval first.

Reply:
talked with design - they are 👍 with this change. will add the larger column style for `spacious` too